pkg/controller/backup_sync_controller.go

```diff
@@ -103,35 +132,36 @@ func (c *backupSyncController) run() {
 	locations = orderedBackupLocations(locations, c.defaultBackupLocation)
 
 	pluginManager := c.newPluginManager(c.logger)
+	defer pluginManager.CleanupClients()
 
 	for _, location := range locations {
 		log := c.logger.WithField("backupLocation", location.Name)
-		log.Info("Syncing backups from backup location")
 
 		backupStore, err := c.newBackupStore(location, pluginManager, log)
 		if err != nil {
 			log.WithError(err).Error("Error getting backup store for location")
 			continue
 		}
 
-		backupsInBackupStore, err := backupStore.ListBackups()
+		syncTime := time.Now().UTC()
+		ok, revision := shouldSync(location, syncTime, backupStore, log)
```

Comment:
Could we not just use `time.Now().UTC()` both inside the `shouldSync` and down below?